### PR TITLE
[Fix] Preserve empty values following short options

### PR DESCRIPTION
**Pull request by oyeong011**
When an empty shell argument follows a short option (`-s ''`), minimist currently sets `s: true` and leaves the empty string in positional arguments. The equivalent long option already consumes the empty value.

Check whether the following argument is undefined, matching the long-option branch. The regression covers inferred and declared strings, short groups, aliases, and unchanged boolean/missing-value behavior.

Validation: failing-first regression; all 177 assertions pass with `npm run tests-only`; `npm run lint` passes (existing warnings); direct CLI invocation of this checkout returns `{ _: ['tail'], s: '' }` for `-s '' tail`. Tested on Node 26.4.0; historical runtime floor and posttest network audit not exercised.

AI disclosure: OpenAI Codex assisted with investigation, implementation, tests, and this description.

### CODE REVIEW

**Comment by shadowspawn** [comment]
This looks like a drive-by AI Pull Request. The account created 30 Pull Requests in a single day.

That said, I think the current behaviour is not as intended. But fixing it might have to be semver major, because although an uncommon case, lots of historical usage!

This is commentary, not a review.

File: test/short.js [hunk 1]
```diff
@@ -67,3 +67,13 @@ test('short and long', function (t) {
 	);
 	t.end();
 });
+
+test('short options consume empty string values', function (t) {
+	t.deepEqual(parse(['-s', '', 'tail']), { s: '', _: ['tail'] });
+	t.deepEqual(parse(['-s', '', 'tail'], { string: 's' }), { s: '', _: ['tail'] });
+	t.deepEqual(parse(['-as', '', 'tail']), { a: true, s: '', _: ['tail'] });
+	t.deepEqual(parse(['-s', ''], { alias: { s: 'string' } }), { s: '', string: '', _: [] });
```
Comment (shadowspawn) [inline test/short.js:75]: This test would be more appropriate with `string` long option name declared in `opts.string`, and `s` alias. 

The current test is a somewhat unlikely looking edge case.

File: test/short.js [hunk 2]
```diff
@@ -67,3 +67,13 @@ test('short and long', function (t) {
...
+	t.deepEqual(parse(['-b', ''], { boolean: 'b' }), { b: true, _: [''] });
+	t.deepEqual(parse(['-s']), { s: true, _: [] });
```
Comment (shadowspawn) [inline test/short.js:77]: This test is of existing behaviour and does not look necessary.

**Comment by shadowspawn** [comment]
For reference, I think this is the relevant line mentioned for the long option (`--foo`) case: https://github.com/minimistjs/minimist/blob/ecfdaea23e7931c0d529c52b743c711c3278a8ce/index.js#L176
